site/docs/v1/tech/apis/scim/_scim-user-response-body.adoc

```diff
@@ -0,0 +1,72 @@
+Because SCIM schemas are completely customizable, the SCIM response returned is generated from the result of the SCIM User object populated by your outgoing response lambda.
+
+The following is an example of a typical SCIM User response body that your SCIM client might expect. However, your outgoing response lambda could generate a completely different SCIM schema is desired.
+
+This is taken from the SCIM Schema RFC describing a SCIM User schema. For the full specification you can find the RFC here:
+
+- https://datatracker.ietf.org/doc/html/rfc7643#page-19
+
+==== Response Body
+[.api]
+[field]#active# [type]#[Boolean]#::
+The state of the SCIM Resource. Typically just mapped from the FusionAuth User.active property.
```

Comment:
How is this different than the enterprise user response? Can you collapse?

Reply:
Or create includes to share?

Reply:
EnterpriseUser is really an extension of User. I can use includes there.

Reply:
Not sure of the best way to handle this since EnterpriseUser is really just a schema extension so I included the User schema in the request anyway.

Reply:
refactor response body into includes as well

Reply:
Should be done now.